src/hooks/useFetcher/index.ts

```
@@ -0,0 +1,53 @@
+      } else {
+        if (json) {
+          setSuccess({ json } as Success<T>)
+        } else {
+          setSuccess({} as Success<T>)
+        }
+      }
+    } catch (err) {
+      setError(getErrorMessage(err))
+      // would be nice to return some error code from BE, so component na read that code and assign it to the correct error msg
+    } finally {
+      if (requestId.current !== newRequestId) return // this is not the latest request
```

Biome (1.9.4)

[error] Unsafe usage of 'return'.

'return' in 'finally' overwrites the control flow statements inside 'try' and 'catch'.

(lint/correctness/noUnsafeFinally)